BankManager.swift

```diff
@@ -6,30 +6,57 @@
 
 import Foundation
 
+enum BankBusiness: CaseIterable {
+    case loan
+    case deposit
+    
+    var string: String {
+        switch self {
+        case .loan:
+            return "대출"
+        case .deposit:
+            return "예금"
+        }
+    }
+}
+
 private class BankClerk {
+    // MARK: - Properties
     var bankWindowNumber: Int
     var isWorking: Bool {
         currentClient != nil
     }
     var currentClient: Client?
-    var workTime: Double = 0.7
-    var totalWorkTime: Double = 0.0
     var finishedClients: Int = 0
     
+    // MARK: - Methods
     func startWork(for client: Client) {
         currentClient = client
-        print("\(client.tag)번 고객 업무 시작")
+        print("\(client.tag)번 \(client.priority.string)고객 \(client.bankBusiness.string)업무 시작")
+        
+        let workTime = Int(self.workTime(bankBusiness: client.bankBusiness) * 1000)
+        DispatchQueue.global().asyncAfter(deadline: .now() + .milliseconds(workTime)) {
+            self.finishWork()
+        }
     }
     
     func finishWork() {
-        totalWorkTime += workTime
         finishedClients += 1
         if let client = currentClient {
-            print("\(client.tag)번 고객 업무 완료")
+            print("\(client.tag)번 \(client.priority.string)고객 \(client.bankBusiness.string)업무 완료")
```

Comment:
열거형의 정의를 위에 제안한대로 변경하면 아마도...
```suggestion
            print("\(client.tag)번 \(client.priority)고객 \(client.bankBusiness)업무 완료")
```
이렇게만 해도 제대로 원하는대로 출력할겁니다.

Reply:
위 열거형에 원시값을 적용하고 출력해본 결과

`client.bankBusiness.rawValue`와 같이 rawValue를 적어주어야 문자열을 출력할 수 있었습니다.

그래서 string 연산 프로퍼티를 정의하고 사용해보았습니다.